server/src/lib/routes/secondfactor/u2f/sign/post.ts

```diff
@@ -11,11 +10,15 @@ import { AuthenticationSessionHandler } from "../../../../AuthenticationSessionH
 import UserMessages = require("../../../../../../../shared/UserMessages");
 import { AuthenticationSession } from "../../../../../../types/AuthenticationSession";
 import { Level } from "../../../../authentication/Level";
+import GetHeader from "../../../../utils/GetHeader";
+import * as Constants from "../../../../../../../shared/constants";
 
 export default function (vars: ServerVariables) {
   function handler(req: express.Request, res: express.Response): BluebirdPromise<void> {
     let authSession: AuthenticationSession;
-    const appId = u2f_common.extract_app_id(req);
```

Comment:
[BREAKING] This was using the Host header but now uses the X-Forwarded-Host header.